src/engine/qcommon/q_math.cpp

```diff
@@ -190,7 +190,7 @@ signed char ClampChar( int i )
 }
 
 // this isn't a real cheap function to call!
-int DirToByte( vec3_t const dir )
+WARN_UNUSED_RESULT int DirToByte( vec3_t const dir )
```

Comment:
IMHO it would be nicer to put the attribute only on the declaration in the header, and not have to additionally put it on the definition.